lib/component-names.js

```diff
@@ -4,12 +4,15 @@
 var md5 = require('md5');
 
 module.exports = {
-  path: function(actualPath) {
+  path: function(actualPath, classicStyleDir) {
```

Comment:
was just wondering if this method could be a little simpler. it's basically doing the same thing twice.. but maybe doing "simpler" could end up being overly annoying and complicated.

```javascript
let terminator = '/';
let pathSegementToRemove = 'components/';

if (actualPath.includes(classicStyleDir)) {
  terminator = '.';
  pathSegementToRemove = 'styles/' + classicStyleDir + '/';
}

return actualPath.substr(0, actualPath.lastIndexOf(terminator)).replace(pathSegementToRemove, '');
```